src/pudl/transform/eia860m.py

```diff
@@ -0,0 +1,91 @@
+"""Module to perform data cleaning functions on EIA860m data tables."""
+
+import pandas as pd
+from dagster import asset
+
+import pudl
+
+logger = pudl.logging_helpers.get_logger(__name__)
+
+
+@asset(io_manager_key="pudl_io_manager")
+def core_eia860m__monthly_generators_changelog(
```

Comment:
This is an interesting naming problem. This table is like our slowly changing dimension tables (`scd`) except it only contains reported records when attributes of an entity actually change and it tracks a `valid_till_date`. This is how [many people model slowly changing dimensions](https://en.wikipedia.org/wiki/Slowly_changing_dimension#Type_2:_add_new_row). Our SCD tables have a record for every time there is a report for an entity even when none of the attributes change.

We could call this asset `core_eia860m__scd_generators` but it wouldn't perfectly align with how we've modeled our other SCD tables. Given we probably aren't going to our current SCD tables' modeling or naming convention maybe we should just stick with "changelog" as the asset type. What do you think @cmgosnell ?

Reply:
yea i wondered this myself. They are somewhat similar but they act very differently in the context  of how the relate to the rest of the db. the `scd` tables have FK relationships to the rest of the db tables while this one definitely does not and should not. I think i would rather keep it clearly different because of this. But I'm open to changing it to `scd`. it definitely is a table of `scd`'s. i'm not sure here.

if we keep do keep `changelog`, it should at least be changed to `core_eia860m__changelog_generators` to be more consistent with the rest.

Reply:
I think the fact it doesn't have FKs with the rest of the db tables and has slightly different modeling from our other SCD tables are important distinctions. Given that I say we go with `core_eia860m__changelog_generators`.